### DISCUSSION TITLE
Which input effects would be good for obscuring my real voice in real time?

### DISCUSSION THREAD

**Discussion by nep2ner** [wwmm/easyeffects#2989] [General]
Which input effects would help obscure my voice in real time? Is it possible with this app? Nothing super fancy, just would rather not give my voice print away to Zoom for free. I know nothing about sound.

**Comment by wwmm** [comment]
The plugin usually used for this task is the `pitch` plugin.

**Reply by violetmage** [reply]
Cursory googling suggests that, short of using neural network powered voice anonymizer, a vocoder would be more appropriate for this application.

**Reply by wwmm** [reply]
If things have to be in a level where encryption has to be used to disguise the person voice in a way it can't be recovered something like a vocoder or more complex will be necessary. But in the level of "nothing super fancy" changing the pitch will be enough to make the voice sound like someone's else.
